### PR TITLE
Hw6

### PR DESCRIPTION
**Pull request by gbielanova**
Functionality can be seen at https://gbielanova.github.io/beetroot_hws_js/hw6.html

### CODE REVIEW

File: js/hw6.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+let circle = new Circle(10);
+document.getElementById("initial_r").innerHTML = circle.radius;
+
+printActualCircle()
+
+// task2
+const MarkerUsedPerSymbol = 0.5;
+
+class Marker {
+    color;
```
Comment (roskyba) [inline js/hw6.js]: We do no need to declare a variable here. Since we are doing it inside `constructor` function

File: js/hw6.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    print(text) {
+        if (!checkTextNotEmpty(text)) {
+            return;
+        }
+
+        let textToPrint = String(text);
+        let symbolsToPrint = textToPrint.replace(' ', '').length;
+
+        if (symbolsToPrint <= this.volume) {
+            this.volume -= symbolsToPrint;
```
Comment (roskyba) [inline js/hw6.js:98]: I think you need to `-= MarkerUsedPerSymbol`

File: js/hw6.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+            this.volume = 0;
+            alert(`${this.color} маркер скінчився!`)
+        }
+        return textToPrint;
+    }
+}
+
+class FillMarker extends Marker {
+    constructor(color) {
+        super(color);
```
Comment (roskyba) [inline js/hw6.js:114]: You can call with `color` and `volume` in `super()`. Since each marker will have color and volume

Comment (gbielanova) [reply js/hw6.js:114]: Am I? Marker class in constructor takes only color and volume is something that predefined in class.

File: js/hw6.js (hunk 1 above)
Comment (gbielanova) [reply js/hw6.js]: Fixed.

File: js/hw6.js (hunk 2 above)
Comment (gbielanova) [reply js/hw6.js:98]: Not exactly, but I do lost MarkerUsedPerSymbol multiplier in a logic. Fixed.

File: js/hw6.js (hunk 3 above)
Comment (roskyba) [reply js/hw6.js:114]: If your will have different markers with volumes and you would like to call parent constructor - than yes

**Review by roskyba — APPROVED** [review]

**Pull request merged**